app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -155,13 +154,6 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
     }
 
     override fun onEngineViewCreated(engineView: EngineView): Disposable? {
```

Comment:
Nit: since this isn't where the `EngineView` is being created anymore, could we rename it to `onEngineViewStarted`?

Reply:
I guess that's not correct either.  What should this be called?

Reply:
That's a good question, why do we need this? We can have a follow-up story to clean up all of our start-up logic, thoughts? @Baron-Severin

Reply:
It's related to the old YouTube focus caching issue.  It looks like we're starting up that caching when the `EngineView` is instantiated, but I don't think we need to be nearly so strict with it (it just needs to be in place before the user has a chance to do anything).  I'm fine putting this into a follow up story, this logic could easily just live in `onStart`.

Reply:
https://github.com/mozilla-mobile/firefox-tv/issues/2053